src/Control/Carrier/State/Strict.hs

```diff
@@ -76,7 +76,7 @@ instance Monad m => Applicative (StateC s m) where
     (s', f') <- f s
     (s'', a') <- a s'
     let fa = f' a'
-    fa `seq` pure (s'', fa)
```

Comment:
1. We do (essentially) the same thing in `>>=`; let’s get that one, too.

2. The `let` binding is redundant now.